src/main/java/org/broadinstitute/hellbender/tools/examples/metrics/multi/ExampleMultiMetricsArgumentCollection.java

```diff
@@ -0,0 +1,24 @@
+package org.broadinstitute.hellbender.tools.examples.metrics.multi;
+
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.metrics.MetricAccumulationLevel;
+import org.broadinstitute.hellbender.metrics.MetricsArgumentCollection;
+
+import java.io.Serializable;
+import java.util.EnumSet;
+import java.util.Set;
+
+/**
+ * Example argument collection for multi-level metrics.
+ */
+public class ExampleMultiMetricsArgumentCollection extends MetricsArgumentCollection implements Serializable {
+
+    public static final long serialVersionUID = 1L;
+
+    @Argument(doc="The level(s) at which to accumulate metrics. Possible values are {ALL_READS, SAMPLE, LIBRARY, READ GROUP}.",
```

Comment:
This Argument should probably be a standard argument collection, since an identical argument is needed for every multilevel metric.